src/Bundle/Resources/config/services.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<container xmlns="http://symfony.com/schema/dic/services"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="http://symfony.com/schema/dic/services
+        https://symfony.com/schema/dic/services/services-1.0.xsd">
+
+    <services>
+        <service id="Zenstruck\Foundry\Manager" public="true">
```

Comment:
Maybe FoundryInitializer ? 

Trying to avoid **Manager, especially a class that’s literally called Manager

Reply:
If I rename to `FoundryInitializer` what should I rename `Factory::manager()` to `Factory::initializer()` - seems a bit weird.

What about `FoundryConfiguration` (or just `Configuration`), "you boot with some configuration and then have access to that configuration"

Alternatively `FoundryContainer` (or just `Container`).